.github/workflows/test-repo-e2e.yml

```
@@ -1,19 +1,28 @@
 name: Integration Test Repository Pipeline
-run-name: Headless E2E test run triggered by adyen-integration-tools-tests ${{inputs.testBranch}}
+run-name: Headless E2E tests - Full Coverage - Test Repo Branch ${{inputs.testBranch}} with Express Checkout ${{inputs.expressBranch}}
```

```suggestion
run-name: E2E tests for Adyen Magento2 Plugin ${{inputs.testBranch}} with Express Checkout Module ${{inputs.expressBranch}}
```

This suggestion is a part of previous comment. Both test cases cover full coverage on different products.

What do you think about this naming convention?